fa25-team-b/README.md

```
@@ -58,21 +74,160 @@ Open a Pull Request to dev. Add your PM and TPM as reviewers.
+---
+
+#### **2. Geographical Range Analysis**
+- Are applicants applying to properties regardless of location?  
+- Are there specific regions with high/low applicant interest?  
+- What regions or major cities are out-of-state applicants applying to?  
+
+---
+
+#### **3. Portfolio Effects**
+
+**Demographic Influence**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)